Objects/unicodeobject.c

```diff
@@ -69,6 +69,8 @@ OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
    in _PyUnicode_ClearInterned(). */
 /* #define INTERNED_STATS 1 */
 
+#define INTERNED_DICT() \
+    _PyRuntime.global_objects.interned
```

Comment:
The parentheses are unnecessary.  However, it may be better to make this an inline function:

```suggestion
static inline void
get_interned_dict(void)
{
    return _PyRuntime.global_objects.interned;
}
```

Furthermore, we may end up moving the interned strings to `PyInterpreterState`.  The above would make that move a little simpler.

Reply:
With the (very) possible move to `PyInterpreterState` in mind, consider taking it a step further right now:

```suggestion
static inline void
get_interned_dict(PyInterpreterState *interp)
{
    return interp->runtime.global_objects.interned;
}
```

Then again, it is probably premature to go that far right now.

Reply:
Changed to static inline functions.